.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,14 +1,20 @@
 ---
 name: Feature request
 about: Suggest an idea for Sonarr
+title: ''
+labels: enhancement
```

Comment:
I actually hate this about the bug template, half the bugs that get filed aren't bugs and new issues that need to be triaged get labeled as bugs straight away. IMO we shouldn't set a label so we can triage properly.

Reply:
I'll update the bug template as well then, incoming pushes

Might I suggest a Support label then so we don't have any non-labeled issues and can reserve non-labeled for items needing triage?  That'll allow us to tag the support GHIs accordingly.

Reply:
Not opposed to a label, but they get insta-closed most of the time.